docs/tutorials/workload-rebalancer.md

```diff
@@ -0,0 +1,263 @@
+all replicas will keep in member2 cluster unchanged.
+
+### Step 3. apply a WorkloadRebalancer to trigger rescheduling.
+
+Assuming you want to trigger the rescheduling of above resources, you can create a new file `workload-rebalancer.yaml`
+and content with the following:
+
+```yaml
+apiVersion: apps.karmada.io/v1alpha1
+kind: WorkloadRebalancer
+metadata:
+  name: demo
+spec:
+  workloads:
+    - apiVersion: apps/v1
+      kind: Deployment
+      name: demo-deploy-1
+      namespace: default
+```
+
+Then run the following command to apply it:
+
+```bash
+kubectl --context karmada-apiserver apply -f workload-rebalancer.yaml
+```
+
+you will get a `workloadrebalancer.apps.karmada.io/demo created` result, which means the API created success.
+
+### Step 4: check the status of WorkloadRebalancer.
+
+Run the following command:
+
+```bash
+$ kubectl --context karmada-apiserver get workloadrebalancer demo -o yaml
+apiVersion: apps.karmada.io/v1alpha1
+kind: WorkloadRebalancer
+metadata:
+  ...
+  creationTimestamp: "2024-05-22T11:16:10Z"
+  name: demo
+  ...
+spec:
+  ...
+status:
+  finishTime: "2024-05-22T11:16:10Z"
+  observedGeneration: 1
+  observedWorkloads:
+    - result: Successful
+      workload:
+        apiVersion: apps/v1
+        kind: Deployment
+        name: demo-deploy-1
+        namespace: default
+```
+
+Thus, you can observe the rescheduling result at `status.observedWorkloads` field of `workloadrebalancer/demo`.
+As you can see, `Deployment/demo-deploy-1` rescheduled successfully.
+
+### Step 5: Observe the real effect of WorkloadRebalancer
+
+You can observe the real replicas propagation status of `Deployment/demo-deploy-1`:
+
+```bash
+$ kubectl --context member1 get po                                          
+NAME                             READY   STATUS    RESTARTS   AGE
+demo-deploy-1-784cd456bf-82kt6   1/1     Running   0          89s
+demo-deploy-1-784cd456bf-k9fhl   1/1     Running   0          89s
+
+$ kubectl --context member2 get po
+NAME                             READY   STATUS    RESTARTS   AGE
+demo-deploy-1-784cd456bf-856rf   1/1     Running   0          9m23s
+```
+
+As you see, rescheduling happened and 2 replicas migrated back to member1 cluster while 1 replica in member2 cluster keep unchanged.
+
+Besides, you can observe a schedule event emitted by `default-scheduler`, such as:
+
+```bash
+$ kubectl --context karmada-apiserver describe deployment demo-deploy-1
+...
+Events:
+  Type    Reason                  Age                From                                Message
+  ----    ------                  ----               ----                                -------
+  ...
+  Normal  ScheduleBindingSucceed  31s                default-scheduler                   Binding has been scheduled successfully. Result: {member2:2, member1:1}
+  Normal  GetDependenciesSucceed  31s                dependencies-distributor            Get dependencies([]) succeed.
```

Comment:
@XiShanYongYe-Chang Please help to check why the `dependencies-distributor` would emit the event, note that the ResourceBinding's `PropagateDeps` is disabled.

Reply:
updated